packages/cli/src/lib/yargs-cli.ts

```diff
@@ -9,8 +9,30 @@ import yargs, {
 } from 'yargs';
 import { PersistConfig, formatSchema } from '@code-pushup/models';
 import { TERMINAL_WIDTH } from '@code-pushup/utils';
+import { version } from '../../package.json';
+import {
+  descriptionStyle,
+  formatNestedValues,
+  formatObjectValue,
+  headerStyle,
+  titleStyle,
+} from './implementation/formatting';
 import { logErrorBeforeThrow } from './implementation/global.utils';
 
+export const yargsDecorator = {
+  'Commands:': `${green('Commands')}:`,
+  'Options:': `${green('Options')}:`,
+  'Examples:': `${green('Examples')}:`,
+  boolean: `${blue('boolean')}`,
+  count: `${blue('count')}`,
+  string: `${blue('string')}`,
+  array: `${blue('array')}`,
+  required: `${blue('required')}`,
```

Comment:
The template strings are unnecessary:

```suggestion
  boolean: blue('boolean'),
  count: blue('count'),
  string: blue('string'),
  array: blue('array'),
  required: blue('required'),
```